generated/nidcpower/nidcpower_service.cpp

```diff
@@ -2385,7 +2385,8 @@ namespace nidcpower_grpc {
       auto status = library_->GetAttributeViSession(vi, channel_name, attribute_id, &attribute_value);
       response->set_status(status);
       if (status == 0) {
-        response->mutable_attribute_value()->set_id(attribute_value);
+        auto session_id = session_repository_->resolve_session_id(attribute_value);
```

Comment:
This change is what fixes the tests covered in #238 in this implementation. The client can't do anything with a `ViSession` in this implementation. They need to resolve to the server allocated session id.